Unix/http/httpclient.c

```diff
@@ -1917,10 +1919,17 @@ Page* _CreateHttpHeader(
     int r;
     char* p;
 
-#define HTTP_HEADER_FORMAT "%s %s HTTP/1.1\r\n" \
-    "Content-Length: %d\r\n"\
-    "Connection: Keep-Alive\r\n" 
+    static const char HTTP_PROTOCOL_HEADER[]     = "HTTP/1.1";
+    static const char HTTP_PROTOCOL_HEADER_LEN = MI_COUNT(HTTP_PROTOCOL_HEADER)-1;
+
+    static const char CONTENT_LENGTH_HEADER[]     = "Content-Length: ";
```

Comment:
We need to actually have storage to point at.  If the client and host strings were always identical, which they are not, we could move their definition to a single files somewhere as a global. Then we wouldn't know who was using them.  

If we just have them in a macro, we would have more copies of them in the object file, plus we would not be able to see them in the debugger.